components/product-details/index.js

```diff
@@ -6,7 +7,7 @@ export const ProductDetails = ({ productName }) => {
     <div className="product-container">
       <img
         className="product-image"
-        src="../../styles/user-female-alt-icon.png"
+        src={path.join('../assets', product.image)}
```

Comment:
I think this needs to be then path.join('./assets',product.image)

Reply:
No since this one level inside component page